packages/server/graphql/private/mutations/addNewFeature.ts

```diff
@@ -35,31 +27,7 @@ const addNewFeature: MutationResolvers['addNewFeature'] = async (
     url,
     id: newFeatureRes.id!
   }
-  const onlineUserIds = new Set()
-  const stream = redis.scanStream({match: 'presence:*'})
```

Comment:
+1 this definitely would be a place where instead of looking for some presence or awareness we should just check for notification subscribers via [PUBSUB CHANNELS](https://redis.io/docs/latest/commands/pubsub-channels/)